packages/cashier/src/components/cashier-locked/cashier-locked.jsx

```diff
@@ -35,219 +61,48 @@ const CashierLocked = ({
     const ask_self_exclusion_max_turnover_set = cashier_validation?.includes('ASK_SELF_EXCLUSION_MAX_TURNOVER_SET');
     const ask_fix_details = cashier_validation?.includes('ASK_FIX_DETAILS');
```

Comment:
Any idea how to simplify these `includes('foo')` parts? Do we use them in other places as well? Should I create a helper function for it?